vyper/utils.py

```diff
@@ -111,8 +111,7 @@ class MemoryPositions:
     MINDECIMAL = 64
     FREE_VAR_SPACE = 128
     FREE_VAR_SPACE2 = 160
-    FREE_LOOP_INDEX = 192
-    RESERVED_MEMORY = 224
+    RESERVED_MEMORY = 192
```

Comment:
This seems like an important change, what happened here?

Reply:
Previously the loop variable in `repeat` (LLL for loop) used a memory location `FREE_LOOPINDEX` and did an mload/mstore every loop iteration to check the value. I moved the loop variable to the stack, so that the memory slot is no longer needed.

Reply:
see https://github.com/vyperlang/vyper/pull/2633/files/4c6b028ba9b273da3559d266152c4997543607af#diff-188b81157988d6a5aa0beb741278b8dd67ed8e9e6fa75554fda4f9f18d4dbde7R212 for the new implementation of `repeat`